LayoutTests/platform/glib/http/tests/resourceLoadStatistics/website-data-removal-for-site-navigated-to-with-link-decoration-expected.txt

```diff
@@ -0,0 +1,35 @@
+Check that website data gets removed after a navigation with link decoration from a prevalent resource.
+
+Before deletion: Client-side cookie exists.
+Before deletion: HttpOnly cookie exists.
+Before deletion: Regular server-side cookie exists.
+Before deletion: LocalStorage entry does exist.
+Before deletion: IDB entry does exist.
+
+After deletion: HttpOnly cookie exists.
+After deletion: Client-side cookie exists.
```

Comment:
Is this expected?

Reply:
Yes, we only delete script-written cookies on Ventura+, so this is expected on the glib port. I considered marking this test as failing on Monterey/glib, but having test coverage on EWS seems valuable even though it is missing some of the desired behavior. I'll let the Linux folks know about this change.

Reply:
Got it. Thanks!